Src/Common/CorrelationIdLookupHelper.cs

```diff
@@ -313,6 +350,20 @@ private void RegisterFailure(string instrumentationKey, Exception ex)
             AppMapCorrelationEventSource.Log.FetchAppIdFailed(this.GetExceptionDetailString(ex));
         }
 
+        /// <summary>
+        /// FetchAppIdFromService failed.
+        /// Registers failure for further action in future.
+        /// </summary>
+        /// <param name="instrumentationKey">Instrumentation key for which the failure occurred.</param>
+        /// <param name="httpStatusCode">Response code from AppId Endpoint.</param>
+        private void RegisterFetchFailure(string instrumentationKey, HttpStatusCode httpStatusCode)
+        {
+            this.failingInstrumenationKeys[instrumentationKey] = new FailedResult(DateTime.UtcNow);
+            
+            const string FetchAppIdResponseErrorFormat = "Fetch AppId failed with response code '{0}'";
+            AppMapCorrelationEventSource.Log.FetchAppIdFailed(string.Format(CultureInfo.InvariantCulture, FetchAppIdResponseErrorFormat, httpStatusCode));
```

Comment:
why do we use event log with the `string.Format`? Please make a dedicated event for it with the argument of response code.